src/interfaces/topup.interface.ts

```diff
@@ -1,41 +1,20 @@
-import { TopUpInputTypeEnum } from 'src/enums/top-up-input-type.enum';
-import { TopUpProviderEnum } from 'src/enums/top-up-providers.enum';
-
-import { IconNameEnum } from '../components/icon/icon-name.enum';
+interface TopUpItemNetwork {
+  code: string;
+  fullName: string;
+  shortName?: string;
+}
 
-export interface TopUpInputInterface {
+export interface TopUpInterfaceBase {
   name: string;
   code: string;
   codeToDisplay?: string;
-  network: string;
-  networkFullName: string;
+  network?: TopUpItemNetwork;
```

Comment:
Please make sure that the app doesn't crash after an upgrade from `development` version because of store schemas differences